src/react-native/13-making-http-requests/fetch-handle-thrown-error.js

```diff
@@ -5,7 +5,7 @@ try {
 
   const data = await response.json()
   const error = response.ok
-    ? null
+    ? undefined
```

Comment:
In the `react-vite` exercises we are still setting as `null` if the response is okay. We should either change those to `undefined` or remove these changes so they align.